keyform/urls.py

```diff
@@ -4,6 +4,7 @@
 from keyform import views
 
 urlpatterns = [
+    url(r'^search', views.SearchView.as_view(), name="search"),
```

Comment:
There's no need for a separate URL & view for this. Integrate it into the HomeView.